proposals/299-ip-failure-count.txt

```diff
@@ -136,7 +136,7 @@ Ticket: https://trac.torproject.org/projects/tor/ticket/27491
    The initial value of the failure points in the scenarios described in this
    section would be:
 
-      * IPv4 Faulure Points: Count the number of IPv6-capable relays
+      * IPv4 Failure Points: Count the number of IPv6-capable relays
 
       * IPv6 Failure Points: Count the number of IPv4-capable relays
```

Comment:
The number of IPv4-capable relays in the consensus is around 7000.
So even if we halve on every connection, we will only be using the local network's feedback after about 10 connections. (7000/2<sup>10</sup> = 7, which is about the same size as 10.)

So we should turn our relay input values into a ratio, rather than adding them directly to the counters. I suggest we create a ratio out of 4, so that we adapt to the local network after 2-4 connections.

Also, for relays in the consensus, we should use their bandwidths to create the ratio, rather than just counting relays. We want IPv6 to be bandwidth-rated.

Reply:
Sounds great. I will use this in my next revision.